pkg/cobrautils/flag/replace.go

```diff
@@ -31,3 +27,15 @@ func IntVarPF(f *pflag.FlagSet, p *int, name, shorthand string, value int, usage
 	f.IntVarP(p, name, shorthand, value, usage)
 	return f.Lookup(name)
 }
+
+// PathVarPF is like PathVarP, but returns the created flag.
+func PathVarPF(f *pflag.FlagSet, p *string, name, shorthand string, value string, usage string) *pflag.Flag {
```

Comment:
this location here is for additional flavors for the standard pflag option types.
For own ones, these flavors should directly be places in the appropriate source file.